lib/ui/mockData/MockLandingPageViewModel.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:farmsmart_flutter/ui/LandingPage.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static String detailText = ("A network and knowledge source for farmers in Kenya");
```

Comment:
why the () ?

Reply:
When i retrieve the Int.message from that i forget to quit the () too, Fixed